apps/frontend/src/app/CreateOrder/CreateOrder.tsx

```diff
@@ -15,33 +14,46 @@ export const CreateOrder: FC = () => {
   const [menu, setMenu] = useState<string>('');
   const [isValid, setIsValid] = useState<boolean>(false);
   const [isValidInput, setIsValidInput] = useState<boolean>(false);
-  const [DeliveryPrice, setDeliveryPrice] = useState<number>(0);
+  const [DeliveryPrice, setDeliveryPrice] = useState<string>('');
 
   const [error, setError] = useState<string | null>(null);
   const history = useHistory();
 
   const onSubmit = (data: CreateOrderDto) => {
-    createOrder({ ...data, orderAt: new Date().toISOString() })
-      .then((params: OrderModel) => history.push(`/`))
+    const newPrice =
+      typeof data.shippingCents === 'string'
+        ? data.shippingCents.replace(/,|\./, '')
+        : '';
+    createOrder({
+      ...data,
+      orderAt: new Date().toISOString(),
+      shippingCents: +newPrice,
```

Comment:
what happens when user types `12,5 `? :D

This is risky and unnecessary. Lets make this input numbers only (higher than `0`)
We'll find or create some custom component which could handle our logic as we want it to

and anyway - I think we should divide this value `/100` because of its `*Cents`